adodbapi/examples/xls_read.py

```diff
@@ -35,7 +35,7 @@
 crsr = conn.cursor()
 sql = "SELECT * from [%s]" % sheet
 crsr.execute(sql)
-for row in crsr.fetchmany(10):
+for row in crsr.fetchmany(10) or ():

```

Comment:
Alternatively, `Cursor._fetch` or `Cursor.fetchmany` could be made to return an empty iterable (`[]` or `()`) when `Cursor.connection` or `cursor.rs` is `None`. Instead of the current behaviour of returning `None`. But that would be a change in behaviour.